runtime/executor/src/main/java/org/apache/nemo/runtime/executor/datatransfer/RunTimeMessageOutputCollector.java

```diff
@@ -44,8 +44,8 @@
 
   private final String taskId;
   private final IRVertex irVertex;
-  private final PersistentConnectionToMasterMap connectionToMasterMap;
-  private final TaskExecutor taskExecutor;
+  private final PersistentConnectionToMasterMap connectionToMasterMap; //NOSONAR
```

Comment:
We decided that it would be better to leave those codes without specifying `//NOSONAR`. Could you delete those comments? I'm really sorry for giving you wrong instructions.